JAICore/jaicore-ml/src/jaicore/ml/cache/Instruction.java

```diff
@@ -0,0 +1,53 @@
+package jaicore.ml.cache;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonSubTypes.Type;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+/**
+ * Instruction class that can be converted into json. Used by {@link ReproducibleInstances}.
```

Comment:
It would be great to give some more details on what kind of information this class is supposed to capture.

Reply:
Apart from that it would also be great to have some documentation on the methods.